### ISSUE TITLE
[Data] EDA, Preprocessing

### ISSUE THREAD

**Issue by taeukkkim** [boostcampaitech2/klue-level2-nlp-14#9]
### 금일 전처리 정리
데이터가 전체적으로 깨끗하며 RE task이기 때문에 subject와 object를 고려하여 함부로 cleansing하기 어려운 issue가 있음. 이를 훼손하지 않는 선에서의 전처리가 필요해 보임.
1. train_test_split을 활용하여 train, validation set 구분. 
    * random_state=42, stratify 적용, train:test = 8:2
2. koelectra 등 한글 위주의 모델을 사용하기 때문에 한자가 들어가 있는 경우 처리 필요
![image](https://user-images.githubusercontent.com/47404628/134876479-2e722e0b-f517-4d5f-ac36-e56e6743d3b9.png)
3. 영어 소문자 처리(lower) 여부
4. 이중 괄호가 되어 있을 경우 내부 괄호 반복 문제 발견
![image](https://user-images.githubusercontent.com/47404628/134891915-d5148363-3433-42e3-992b-147aff5407e8.png)
![image](https://user-images.githubusercontent.com/47404628/134891784-b577cedb-7998-4c6a-826e-d0a48fa5c98d.png)

### 적용해볼만한 처리기법
1. kss 라이브러리 사용
2. 한자 -> 한글 처리 번역툴 이용
* 참고: https://github.com/kakaobrain/pororo/blob/7d05a75e8062b00e6b65364b8ec6c52b6293ab07/pororo/tasks/word_translation.py#L9
3. k-fold cross validation
